doc/gov/chapters/chapter01.md

```diff
@@ -11,18 +11,18 @@
 <a name="1.1"></a>
 ## 1.1 Who We Are
 
-The founding members of the Common NFVI Telco Task Force (CNTT) were AT&T, Vodafone, Verizon, Deutsche Telecom, Orange, and China Mobile. Soon thereafter, additional telco operator and vendor partner participants began to steadily join the Task Force. CNTT reached its first major milestone of gaining the sponsorship and support of GSMA and The Linux Foundation Networking in Summer 2019. It now has over thirty operators and partners (VNF suppliers, third-party integrators, hw/sw suppliers) in its member commmunity and continue to grow. Together they are leading the industry in creating a common Network Function Virtualization Infrastructure in the form of reference model and reference architecture definitions, as well as collaborating with OPNFV and other open source communities to operationalize and support reference implementations and reference certification platforms. 
+The founding members of the Cloud iNfrastructure Telco Task Force (CNTT) were AT&T, Vodafone, Verizon, Deutsche Telecom, Orange, and China Mobile. Soon thereafter, additional telco operator and vendor partner participants began to join the Task Force. CNTT reached its first major milestone when it gained sponsorship and support of the GSMA and Linux Foundation Networking in Summer 2019. As of Summer 2020, there are over thirty operators and partners (VNF suppliers, third-party integrators, hw/sw suppliers) in its member commmunity and these numbers continue to grow. Together th CNTT community is leading the industry in creating a common infrastructure reference platform in the form of reference model and reference architecture definitions to better support Network Function Virtualization (NFV) applications for the Telecom industry as a whole. The working group is collaborating with OPNFV and other open source communities to operationalize and support reference implementations and reference certification platforms. 
```

Comment:
Nitpick: `Together th CNTT` -> `Together the CNTT`

Reply:
done